packages/server/src/cli.ts

```diff
@@ -166,6 +166,24 @@ if (isCli) {
 		password = buffer.toString("hex");
 	}
 
+	// If CLI password was provided, obsfucate password from process title
+	if (options.password) {
+		const parts = [process.title];
+		for (let i = 2; i < process.argv.length; i++) {
+			if (process.argv[i].startsWith("--password=")) {
+				parts.push(process.argv[i].replace(/=.*/, "=****"));
+			} else if (process.argv[i] === "--password") {
```

Comment:
My bad, I didn't include `-P` in my example but we'll need that as well. No need to check for `-P=` though since `=` isn't used with short options.